worlds/inscryption/Options.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+
+from Options import Toggle, Choice, Option, PerGameCommonOptions
+import typing
+
+
+class DeathLink(Toggle):
+    """Activate DeathLink"""
+    display_name = "DeathLink"
```

Comment:
Any specific reason you're not just importing DeathLink from Options?

Reply:
Didn't know that was a thing. I'll look into it.

Reply:
Changed to DeathLinkMixin. Gonna leave this as unresolved in case I didn't do it the right way.